lms/djangoapps/save_for_later/api/v1/views.py

```diff
@@ -127,6 +127,9 @@ def post(self, request):
         if get_email_validation_error(email):
             return Response({'error_code': 'incorrect-email'}, status=400)
 
+        if not program_uuid:
```

Comment:
Is this being handled in prospectus?

Reply:
handled. but someone can access api directly.